data/json/items/comestibles/mushroom.json

```diff
@@ -99,77 +81,49 @@
     "type": "COMESTIBLE",
     "id": "dry_mushroom_magic",
     "name": "dried hallucinogenic mushroom",
-    "weight": 15,
-    "color": "brown",
-    "spoils_in": "360 days",
-    "comestible_type": "FOOD",
-    "symbol": "%",
-    "quench": -1,
-    "calories": 69,
+    "copy-from": "dry_mushroom",
     "description": "A hallucinogenic mushroom which has been dehydrated for storage.  Will still cause hallucinations if eaten.",
-    "price": 180,
-    "material": "mushroom",
-    "volume": 1,
-    "flags": [ "EDIBLE_FROZEN" ],
-    "charges": 4,
-    "fun": 1
+    "charges": 4
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.